### PR TITLE
Add zenMode.hideActivityBar option

### PR DESCRIPTION
**Pull request by lifez**
Connected to #23611

### CODE REVIEW

**Comment by lifez** [comment]
Could anyone guide me how to test this function

File: src/vs/workbench/electron-browser/main.contribution.ts [hunk 1]
```diff
@@ -298,6 +298,11 @@ configurationRegistry.registerConfiguration({
 			'default': true,
 			'description': nls.localize('zenMode.hideStatusBar', "Controls if turning on Zen Mode also hides the status bar at the bottom of the workbench.")
 		},
+		'zenMode.hideActivityBar': {
+			'type': 'boolean',
+			'default': true,
+			'description': nls.localize('zenMode.hideActivityBar', "Controls if turning on Zen Mode also hides the activitybar bar at the left of the workbench.")
```
Comment (isidorn) [inline src/vs/workbench/electron-browser/main.contribution.ts]: Please use 'activity bar' not 'activitybar' in the description

File: src/vs/workbench/electron-browser/workbench.ts [hunk 2]
```diff
@@ -1109,7 +1110,10 @@ export class Workbench implements IPartService {
 			this.setPanelHidden(true, true).done(undefined, errors.onUnexpectedError);
 			this.setSideBarHidden(true, true).done(undefined, errors.onUnexpectedError);
 
-			this.setActivityBarHidden(true, true);
+			this.setActivityBarHidden(false, false);
```
Comment (isidorn) [inline src/vs/workbench/electron-browser/workbench.ts]: This line is not needed. `this.setActivityBarHidden(false, false);`
If the setting is not `true` we simply should not do anything

**Comment by isidorn** [comment]
@lifez thank you for your PR, I have commented twice in the commit. Please address those comments and then we can merge in this PR.

**Comment by lifez** [comment]
@isidorn done!

**Comment by isidorn** [comment]
@lifez thank you I have merged in the PR and it looks good.
I have only fixed a typo via this commit 6cbd65381

**Pull request merged**